examples/kube-stack-private/README.md

```diff
@@ -0,0 +1,103 @@
+
+##### Install Kubernetes Using Kubespray
+
+You are now ready to use Kubespray to install Kubernetes in the cluster you just created. Follow the following steps:
+1. Use `pip` to install the following Python packages: `boto3`,`os`,`json`,`argparse`,`netaddr`
+2. If you don't already have it installed, use pip to install [`ansible`](https://github.com/ansible/ansible)
+3. `git clone` the [Kubespray repository](https://github.com/kubernetes-incubator/kubespray/) from the Kubernetes Incubator to your local machine and then go into the created directory
+4. We need to create a directory where we can define the specific parameters for our own cluster. In the `inventory` directory, copy the sample directory to a directory named after whatever you want to name your cluster, as follows:
+```bash
+cp -R inventory/sample/ inventory/mycluster
+```
+5. One set of parameters we need to create our how to allocate the different EC2 for different purposes in the Kubernetes cluster. The Terraform setup has added tags to each instance to define its role in the cluster. We will use a Python script to dynamically create the inventory list (to read more about this, see [here](https://github.com/kubernetes-incubator/kubespray/blob/master/docs/aws.md). Copy the `kubespray-aws-inventory.py` Python script into the directory you just created:
+```bash
+cp contrib/aws_inventory/kubespray-aws-inventory.py inventory/mycluster/
+```
+6. You need to define the following three variables for this script to work properly:
+```bash
+export AWS_ACCESS_KEY_ID="xxxxx"
+export AWS_SECRET_ACCESS_KEY="yyyyy"
+export REGION="<region where you put the cluster>"
+```
+7. To test that you properly set things up in the previous step run the following command. It should print out your inventory in the terminal, listing all the nodes along with what role(s) they have in the cluster: `etcd`,`master` and/or `node`
+```bash
+python inventory/mycluster/kubespray-aws-inventory.py --list
+```
+8. Next you need to set some parameters in the file `inventory/mycluster/group.vars/all.yml`. Uncomment these variables and/or set the values as directed below. Note that these variables are scattered throughout the file so you have to look for them. Note as well you should review the entire file since there may be other variables you might like to change:
+```
+bootstrap_os: coreos
+apiserver_loadbalancer_domain_name: "<see note below on how to find elb name which should put here between quotes>"
+# since we use an external load balancer we need to set local lb to false
+loadbalancer_apiserver_localhost: false
+cloud_provider: aws
+etcd_memory_limit: 0
+```
+9. You can find the DNS name for the API server load balancer by running `terraform show` and searching for `module.kube-cluster.aws_elb.kube-controllers`. One of the parameters listed under this is `dns_name`. Copy the value into the appropriate place above.
```

Comment:
is there an output with that info, from the example env?

Reply:
Fixed- added properly modifed sample output.